frame/sudo/src/lib.rs

```diff
@@ -195,7 +203,7 @@ pub mod pallet {
 		/// ## Complexity
 		/// - O(1).
 		#[pallet::call_index(2)]
-		#[pallet::weight({0})] // FIXME
+		#[pallet::weight(T::WeightInfo::set_key())] // FIXME
```

Comment:
```suggestion
		#[pallet::weight(T::WeightInfo::set_key())]
```
It is fixed now :smile: